<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1217</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Проектный расчет двухступенчатой ракеты</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>60</y>
      <width>641</width>
      <height>481</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>false</underline>
     </font>
    </property>
    <property name="toolTipDuration">
     <number>0</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>271</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>ВХОДНЫЕ ПАРАМЕТРЫ</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>81</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>-4.3</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>81</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>0.0344</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Разворот</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Вывод</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_10">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>81</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>1.7</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_11">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>110</y>
       <width>81</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>1.1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Тяговооруженность 1СТ</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Тяговооруженность 2 СТ</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>610</y>
      <width>1171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="toolTipDuration">
     <number>0</number>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::NoDragDrop</enum>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::CustomDashLine</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>100</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>135</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>30</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>30</number>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <row>
     <property name="text">
      <string>Первый РБ</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Второй РБ</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Длина бака &quot;О&quot;, м</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Длина бака &quot;Г&quot;, м</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Объем бака &quot;О&quot;, м^3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Объем бака &quot;Г&quot;, м^3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Масса &quot;О&quot;, кг</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Масса &quot;Г&quot;, кг</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Длина ХО, м</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Тяга ДУ, тс</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>1</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
     <property name="foreground">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
     <property name="icon">
      <iconset theme="applications-internet">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </item>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>271</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>ДАННЫЕ РАСЧЕТА</string>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>81</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Скорость, м/c</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>81</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Высота, м</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Дальность полета МБ, м</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_7">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>110</y>
       <width>81</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Время полета I ст, с</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>190</y>
       <width>81</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>150</y>
       <width>81</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Макс скор напор, кг/мс2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Угол траектории (2), град</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Угол траектории (1), град</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_9">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>270</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_8">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>230</y>
       <width>81</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Кон. угол атаки, град</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_12">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>350</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Угол траектории (3), град</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_13">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>310</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>231</width>
      <height>491</height>
     </rect>
    </property>
    <property name="title">
     <string>              ВЫХОДНЫЕ ПАРАМЕТРЫ</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>215</width>
       <height>446</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="heightButton">
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="text">
         <string>Высота</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="distanceButton">
        <property name="text">
         <string>Дальность</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="trjButton">
        <property name="text">
         <string>Траектория</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="velButton">
        <property name="text">
         <string>Скорость</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="alphaButton">
        <property name="text">
         <string>Угол атаки</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="angleButton">
        <property name="text">
         <string>Угол траектории</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="vhButton">
        <property name="text">
         <string>Скоростной напор</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="centerButton">
        <property name="text">
         <string>Центр масс</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_11">
        <property name="text">
         <string>Тяга</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="massButton">
        <property name="text">
         <string>Масса</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_14">
        <property name="text">
         <string>Продольная перегрузка</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_15">
        <property name="text">
         <string>Фокус</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="staticButton">
        <property name="text">
         <string>Стат момент</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="YZmomentButton">
        <property name="text">
         <string>Поперечный центробежный момент</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="XmomentButton">
        <property name="text">
         <string>Продольный центробежный момент</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1217</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Расчет</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="action_6"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_4"/>
   </widget>
   <addaction name="menu_2"/>
   <addaction name="menu"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>Трасса полета</string>
   </property>
  </action>
  <action name="action_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>МЦИХ</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Динамика</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_6">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Углы</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
